src/components/MonthCalendar.tsx

```diff
@@ -0,0 +1,83 @@
+import { FormattedDate } from 'react-intl';
+
+import { ZetkinEvent } from '../types/zetkin';
+
+interface MonthCalendarProps {
+    month?: number;
+    year?: number;
+    events: ZetkinEvent[];
```

Comment:
I'm surprised the linter isn't complaining that these are not in alphabetical order. Maybe it's a rule that we haven't enabled yet.